AstSemantics.md

```diff
@@ -156,53 +149,12 @@ when considering alignment.
 If the effective address of a memory access is a multiple of the alignment
 attribute value of the memory access, the memory access is considered *aligned*,
 otherwise it is considered *misaligned*. Aligned and misaligned accesses have
-the same behavior. Alignment affects performance as follows:
-
- * Aligned accesses with at least natural alignment are fast.
- * Aligned accesses with less than natural alignment may be somewhat slower
-   (think: implementation makes multiple accesses, either in software or
-    in hardware).
- * Misaligned access of any kind may be *massively* slower
-   (think: implementation takes a signal and fixes things up).
-
-Thus, it is recommend that WebAssembly producers align frequently-used data
-to permit the use of natural alignment access, and use loads and stores with
-the greatest alignment values practical, while always avoiding misaligned
-accesses.
```

Comment:
The discussion of the affects of alignment wasn't intended as rationale; it's guidance to developers to know what to expect from alignment fields, and to implementors to know what they're expected to do with alignment fields.